app/post.go

```diff
@@ -105,6 +105,10 @@ func CreatePost(post *model.Post, channel *model.Channel, triggerWebhooks bool)
 
 	post.Hashtags, _ = model.ParseHashtags(post.Message)
 
+	if err := FillInPostProps(post, channel); err != nil {
```

Comment:
I think we're going to want to load test with this added in. It's running a fair bit of extra processing before the post is made since it might involve multiple DB calls